### ISSUE TITLE
Add the stuff from Type Lumina

### ISSUE THREAD

**Issue by Siuolplex** [Arathain/The-Arcpocalypse#1]
This is a checklist mostly so I know what I plan to add in a PR

- Add the new Necos (Possibly make transformation items as well) [Done]
  - Neco-Ciel
  - Neco-Akiha
  - Neco-Kohaku
  - Neco-Hisui

- Add some of Neco Arc's moves
   - Oh God I'm adding a phone AND a gacha game to Minecraft
   - Let you go to a alligator garden
   - Let you evade a ton
   - Let you travel to alternate timelines

**Comment by Arathain** [comment]
laser eyes and rocket are already in btw

**Comment by Siuolplex** [comment]
Oh dam nice

**Comment by Lhruggles** [comment]
Yeah, neco arc’s phone/summons are definitely important imo. Probably add each of the apps she uses in type lumina. I also think the implementation of FGO should be special, spending emeralds to get mapo tofu, curry, and black keys, with a chance of summoning artoria/mash as mobs to help you out.

**Comment by Siuolplex** [comment]
I'm gonna have a field day making the models for the needed servants

**Comment by Lhruggles** [comment]
I bet they’ll look great! This mod has a ton of potential, I really hope it’ll succeed.

**Comment by Siuolplex** [comment]
I'll add the other Necos first before I add the phone as the phone will be relatively difficult for me

**Comment by Siuolplex** [comment]
Alright that's done a lot is next

**Comment by Siuolplex** [comment]
Something something bones and swords
